test/test_event.rb

```diff
@@ -402,4 +402,115 @@ def setup
       }
     end
   end
+
+  class CompressedMessagePackEventStreamTest < ::Test::Unit::TestCase
+    include Fluent
+    include DeepCopyAssertion
+    include Fluent::Plugin::Compressable
+
+    def setup
+      time = Engine.now
+      @times = [Fluent::EventTime.new(time.sec), Fluent::EventTime.new(time.sec + 1)]
+      @records = [{ 'k' => 'v1', 'n' => 1 }, { 'k' => 'v2', 'n' => 2 }]
+      @packed_record = ''
+      @entries = ''
+      @times.zip(@records).each do |_time, record|
+        v = ''
+        v << [_time, record].to_msgpack
+        @packed_record += v
```

Comment:
`@packed_record << [_time, record].to_msgpack` looks to work well, and it's simple.